sklearn/tests/test_random_projection.py

```diff
@@ -24,7 +24,7 @@
 
 all_SparseRandomProjection: List[Any] = [SparseRandomProjection]
 all_DenseRandomProjection: List[Any] = [GaussianRandomProjection]
-all_RandomProjection = set(all_SparseRandomProjection + all_DenseRandomProjection)
+all_RandomProjection = all_SparseRandomProjection + all_DenseRandomProjection
```

Comment:
This was done to avoid a `pytest-xdist` error about workers collecting different tests (different order).

It does not seem that crucial that all_RandomProjection is a set in this file.